pc

```diff
@@ -310,9 +310,10 @@ elif [[ "$CMD" == "test-backend" ]]; then
     PYTEST_ARGS=$*
   fi
   RUN='docker stop pc-postgres-pytest;
-    docker rm pc-postgres-pytest;
-    docker-compose up -d postgres-test;
-    docker exec pc-flask bash -c "cd /opt/services/flaskapp/src/ && rm -rf static/object_store_data/thumbs/* && PYTHONPATH=. pytest --color=yes -rsx -v $PYTEST_ARGS"'
+        docker rm -v pc-postgres-pytest;
+        docker-compose up -d postgres-test;
+        while ! docker-compose logs postgres-test 2> /dev/null | grep -q "listening on IPv4 address"; do echo "waiting for test-database"; sleep 0.5; done;
```

Comment:
Je veux bien plus de précisions à l'oral de cette nouvelle ligne.

Reply:
Que penses tu d'utiliser **pg_isready** à la place de grep dans les logs ? "pg_isready -h localhost -p 5433" 
https://www.postgresql.org/docs/10/app-pg-isready.html
Et de mettre tout ça dans une boucle for pour ne pas rester bloquer si le conteneur ne démarre jamais ?

Reply:
j'ai mis le pg_isready, mais pour la boucle for je vois pas bien ce que tu voulais dire